packages/features/auth/lib/dub.ts

```diff
@@ -3,3 +3,13 @@ import { Dub } from "dub";
 export const dub = new Dub({
   token: process.env.DUB_API_KEY,
 });
+
+// fetch Dub customer using their external ID (ID in our database)
+export const getDubCustomer = async (userId: string) => {
+  const customer = await dub.customers.list({
```

Comment:
Does this method always return an array? What if a self hoster doesn't have dub setup?

Reply:
oh good catch, added a `process.env.DUB_API_KEY` check 👍

Reply:
good change 👍